.gitlab-ci.yml

```diff
@@ -16,6 +16,7 @@ workflow:
     - if: '$CI_EXTERNAL_PULL_REQUEST_IID'
     - if: '$CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH'
     - if: $CI_PIPELINE_SOURCE == "web"
+    - if: $CI_COMMIT_TAG
```

Comment:
Looks fine for me. Although the tagging will trigger  the entire workflow, but it won't delay any ongoing PR. 
A nitpick, can we add a comment here to remind that this rule is for triggering the image build?